skycoin/skywire
# Clean logs

Did you run `make format && make check`?
yes

Fixes #1234 

 Changes:	
- Changed repetitive logs to trace
- Changed Extra-informative logs to debug
- Added Info logs where needed
- Fixed some closing logs
- Set `Stdout` of proc to Debug
- Set `Stderr` of proc to Error
- Removed logs from apps
- Changed info logs to fmt.Print in apps
- Changed error logs to print() in apps
- Removed unnecessary lock and unlock logs

How to test this PR:
1. Checkout [#skywire-utilities-25](https://github.com/skycoin/skywire-utilities/pull/25)
2. Checkout [#dmsg-176](https://github.com/skycoin/dmsg/pull/176)
3. In dmsg in the file `go.mod` uncomment `replace github.com/skycoin/skywire-utilities => ../skywire-utilities` and run `make dep`
4. In skywire in the file `go.mod` uncomment `replace github.com/skycoin/skywire-utilities => ../skywire-utilities` and `replace github.com/skycoin/dmsg => ../dmsg` and run `make dep; make build`
5. Set the `log_level` in the config to `info`, `debug`, and `trace` and test the functionalities and apps of the visor
6. Check logs
